.github/workflows/lint.yml

```diff
@@ -30,7 +30,16 @@ jobs:
         run: npm ic
 
       - name: Run Tests
-        run: npm run unit
+        run: npm test || npm test || npm test
```

Comment:
This is here to automatically try running the tests again, to get another server.
Lets remove this multiple testing for now, evaluate how often that might happen, and we update it later if needed.
So, replace:
`run: npm test || npm test || npm test`
by
`run: npm test`